### DISCUSSION TITLE
Difficulties to creating the EKS cluster

### DISCUSSION THREAD

**Discussion by saxenachaitanya** [particuleio/teks#33] [Q&A]
Hello Team ,

I'm trying to create the EKS cluster but I'm confused how to create the EKS cluster I have read your docs but still I can't able to figure out how we can I create the eks cluster. If you had any videos regarding to create the EKS cluster that would be soo helpful

**Comment by ArchiFleKs** [comment]
@chaitanyasaxena246 do you have the terragrunt.hcl configuration you are trying to use ? And also your global configuration ?

**Comment by saxenachaitanya** [comment]
I have tried it but I can't able to figure out how can I run the EKS cluster . Also did you create any high level architect soo that we can see what we are creating

**Reply by ArchiFleKs** [reply]
Not yet lack of time

**Comment by ArchiFleKs** [comment]
Could you retry with latest release ?
